File: crates/aft/tests/fixtures/extract_function/sample.py
import os

BASE_DIR = "/tmp/data"

def process_data(items, prefix):
    filtered = [item for item in items if len(item) > 0]
    mapped = [prefix + item for item in filtered]
    result = ", ".join(mapped)
    print(result)
    return result

def simple_helper(x):
    doubled = x * 2
    added = doubled + 10
    return added

def void_work(name):
    greeting = "Hello, " + name
    print(greeting)

File: crates/aft/tests/fixtures/extract_function/sample.ts
import { readFile } from "fs";

const BASE_URL = "https://api.example.com";

function processData(items: string[], prefix: string): string {
  const filtered = items.filter(item => item.length > 0);
  const mapped = filtered.map(item => prefix + item);
  const result = mapped.join(", ");
  console.log(result);
  return result;
}

function simpleHelper(x: number): number {
  const doubled = x * 2;
  const added = doubled + 10;
  return added;
}

function voidWork(name: string): void {
  const greeting = "Hello, " + name;
  console.log(greeting);
}
